lab_7_llm/assets/main.js

```diff
@@ -0,0 +1,28 @@
+let inferSample = async (premise, hypothesis, result) => {
+    await fetch('/infer', {
+        method: 'POST',
+        headers: {
+            'Content-type': 'application/json'
+        },
+        body: JSON.stringify({
+            question: premise.value,
+            hypothesis: hypothesis.value
+        })
+    }).then(res => {return res.json()})
```

Comment:
I think this is the third time I've seen this code.
`async \ await` function was invented as the development of promises and getting rid of `then` and promise hell